js/interactive-guides/modules/content-manager.js

```diff
@@ -575,7 +583,8 @@ var contentManager = (function() {
         Check if all of the instructions for a given step are complete.
         Input: {stepName} Step name
     */
-    var enableNextWhenAllInstructionsComplete = function(stepName) {
+    var enableNextWhenAllInstructionsComplete = function(step) {
+        var stepName = step.name;
         var isComplete = true;
         var stepInstruction = __getStepInstruction(stepName);
         if(stepInstruction.instructions.length > 0){
```

Comment:
I know this is not a new line of code. What if the step has no instruction set. The if statement should change to 
if (stepInstruction.instructions && stepInstrcution.instructions.length > 0)

Reply:
Done